### PR TITLE
feat: Lightweight Styling sample page

### PR DESCRIPTION
**Pull request by silviuo**
GitHub Issue (If applicable): #824 

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the new behavior?

Created a sample page for lightweight styling override:

image

Material
image
image
image

Fluent
image
image
image

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested on iOS.
- [x] Tested on Wasm.
- [x] Tested on Android.
- [x] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/LightweightStylingSamplePage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.Theming, "Lightweight Styling", Description = "Lightweight styling enables the customization of a control's appearance by adjusting individual keys rather than completely redefining its style.", DocumentationLink = "https://material.io/components/cards")]
```
Comment (agneszitte) [inline Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/LightweightStylingSamplePage.xaml.cs:5]: DocumentationLink is incorrect here @silviuo

Comment (silviuo) [reply Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/LightweightStylingSamplePage.xaml.cs:5]: hi @agneszitte , is this the correct link?

https://learn.microsoft.com/en-us/windows/apps/design/style/xaml-styles#lightweight-styling

Thanks!

**Comment by silviuo** [comment]
I pushed the new link and I also included a master rebase

**Review by kazo0 — APPROVED** [review]

**Comment by kazo0** [comment]
Great job!

**Comment by kazo0** [comment]
@Mergifyio backport legacy/4x

**Pull request merged**